homework-17/jest-test/calculatorJT.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { Calculator } from "../src/calculator";
+
+let calculator: any;
+
+const VALIDATION_ERROR = "The result is not as expected!";
+
+describe("Tests for division operations", () => {
+    beforeAll(() => {
+      calculator = new Calculator();
+    });
+    test("Should divide two numbers correctly", () => {
+        expect(calculator.division(25, 5)).toBe(5);
+    });
+    test("Should divide of 0 by a number correctly", () => {
```

Comment:
Please delete "by" particle from the test name:
`Should divide 0 by a number correctly`
The same is for other methods.